From 384fc9e6f8346d82182af357d9e1bea138e09cf6 Mon Sep 17 00:00:00 2001
From: vdanix <vishwanathx.dani@intel.com>
Date: Mon, 26 Sep 2022 14:31:45 +0530
Subject: [PATCH] Fix CtsGraphicTest testOpenGlEsDeqpLevel

TestCase checking pm deqp level
Added android.software.opengles.deqp.level feature list

Tracked-On: OAM-104025
Signed-off-by: vdanix <vishwanathx.dani@intel.com>

diff --git a/caas/device.mk b/caas/device.mk
index d81a2f2..aea3a37 100644
--- a/caas/device.mk
+++ b/caas/device.mk
@@ -456,6 +456,9 @@ PRODUCT_PROPERTY_OVERRIDES += \
 PRODUCT_COPY_FILES += \
     frameworks/native/data/etc/android.hardware.opengles.aep.xml:vendor/etc/permissions/android.hardware.opengles.aep.xml
 
+PRODUCT_COPY_FILES += \
+    frameworks/native/data/etc/android.software.opengles.deqp.level-2020-03-01.xml:vendor/etc/permissions/android.software.opengles.deqp.level.xml
+
 # GLES version
 PRODUCT_PROPERTY_OVERRIDES += \
    ro.opengles.version=196610
-- 
2.37.3

